### DISCUSSION TITLE
Global Dynamic Import hints are not working as expected

### DISCUSSION THREAD

**Discussion by saravanan10393** [webpack/webpack#16281] [Q&A]
After the release of [Global dynamic import hint](https://webpack.js.org/configuration/module/#moduleparserjavascriptdynamicimportprefetch), i thought to remove all of my import hint inside source code and move it globally to the config file. But no module preloaded. 

Am i missing something. 

here is my webpack config

```js
  module: {
    parser: {
      javascript: {
        dynamicImportMode: "eager",
        dynamicImportPreload: true
      }
    }
  }
```

**Comment by alexander-akait** [comment]
Can you provide reproducible example?

**Comment by saravanan10393** [comment]
@alexander-akait , here is the stackblitz url to reproduce this issue
[https://stackblitz.com/edit/github-99hz2z](https://stackblitz.com/edit/github-99hz2z)

**Reply by alexander-akait** [reply]
Firstly - are you sure you need `dynamicImportPreload`? Because browser always load them and it is bad for perf, really, I recommend to use `dynamicImportPrefetch`, but better apply `RegExp` here and don't do it for all asyn chunks

You can read more infromation here https://github.com/webpack/webpack/issues/8342
